letsencrypt-auto-source/pieces/bootstrappers/rpm_python3_legacy.sh

```diff
@@ -2,6 +2,17 @@
 # number must be increased.
 BOOTSTRAP_RPM_PYTHON3_LEGACY_VERSION=1
 
+EnablePython36SCL() {
+  set +e
```

Comment:
Setting this early has the side effect that if we return early, `set -e` isn't run and certbot-auto will not crash on errors.

Reply:
I fixed that.